templates/zerver/help/import-from-gitter.md

```diff
@@ -1,117 +1,193 @@
 
-- This tool doesn't translate Gitter's Markdown format into Zulip
-  format Markdown (there are a few corner cases where the syntax is
-  different).  Additionally, Gitter's issue mention syntax isn't translated.
+- Zulip's import tool doesn't translate Gitter's Markdown format into Zulip's
+  Markdown format (there are a few corner cases where the syntax is different).
+  Additionally, Gitter's issue mention syntax isn't translated.
+
+- Message edit history is not imported.
+
+[grant-admin-access]: https://zulip.readthedocs.io/en/latest/production/management-commands.html#grant-administrator-access)
+[gitter-api-user-data]: https://developer.gitter.im/docs/user-resource
+
+## Get your organization started with Zulip
+
+Once the import process is completed, you will need to:
+
+{start_tabs}
+
+1. [Configure the settings for your organization](/help/customize-organization-settings),
+   which are not exported. This includes settings like [email
+   visibility](/help/restrict-visibility-of-email-addresses), [message editing
+   permissions](/help/configure-message-editing-and-deletion#configure-message-editing-and-deletion_1),
+   and [how users can join your organization](/help/restrict-account-creation).
+
+1. [Configure user roles](/help/change-a-users-role). Only organization owners
+   and administrators can do this.
+    * If you [import into Zulip Cloud](#import-your-data-into-zulip), you will
+    specify the user whose account will have the owner role when you request the
+    import.
+    * If you self-host, you can follow the Zulip documentation on [making a user an
+    organization owner from the terminal][grant-admin-access] to mark the appropriate
+    users as organization owners.
+
+1. All users from your previous workspace will have accounts in your new Zulip
+   organization. However, you will need to let users know about their new
+   accounts, and [how they will log in for the first time
+   ](#how-users-will-log-in-for-the-first-time).
+
+1. Share the URL for your new Zulip organization, and (recommended) the [Getting
+   started with Zulip guide](/help/getting-started-with-zulip).
+
+1. Migrate any [integrations](/integrations).
+
+{end_tabs}
+
+## How users will log in for the first time
+
+When user accounts are imported, users initially do not have passwords
+configured.
+
+!!! tip ""
+
+    For security reasons, passwords are never exported.
+
+When you create your organization, users will immediately be able to log in
+without a password using GitHub as the default
+[authentication method](/help/configure-authentication-methods).
+
+Users whose accounts have been imported will have to use GitHub authentication
+to log in to Zulip, and will need to [change their Zulip email address
+](/help/change-your-email-address) in order to receive [email notifications
+](/help/email-notifications). Then users can reset their own passwords by
+following the instructions on your Zulip organization's login page.
+
```

Comment:
This section feels a bit out of order to me. How about something like this?
-----

## How users will log in for the first time

When you create your organization, users will immediately be able to log in
without a password using GitHub as the
[authentication method](/help/configure-authentication-methods). Once they log in, users whose accounts have been imported will need to [change their Zulip email address
](/help/change-your-email-address) in order to receive [email notifications
](/help/email-notifications).

!!! warn ""
    A user's email notifications will not work until they update the email associated with their Zulip account.

When user accounts are imported, users initially do not have passwords
configured. Users can reset their own passwords by
following the instructions on your Zulip organization's login page.

!!! tip ""

    For security reasons, passwords are never exported.